erpnext/quality_management/doctype/quality_review/quality_review.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2018, Frappe and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Quality Review', {
+	onload: function(frm){
+		if(frm.doc.date == null){
+			frm.set_value("date", frappe.datetime.get_today());
+		}
+		if(frm.doc.measurable == "Yes"){
+			frm.fields_dict.values.grid.docfields[1].hidden = 0;
+			frm.fields_dict.values.grid.docfields[2].hidden = 0;
+			frm.fields_dict.values.grid.docfields[3].hidden = 0;
+			frm.fields_dict.values.grid.docfields[4].hidden = 1;
+			frm.refresh();
+			$("div[data-fieldname='achieved']").show();
```

Comment:
this looks like a hack! There must be a better way to fix this.

Reply:
```$("div[data-fieldname='achieved']").show();```

Actually we can not hide(toggle) the columns from grid view by using  ```depends_on``` or ``` in_list_view``` parameter so, used jQuery. I have discussed it with faris. he said to fix it.

Reply:
```$("div[data-fieldname='achieved']").show();```

Actually we can not hide(toggle) the columns from grid view by using  ```depends_on``` or ``` in_list_view``` parameter so, used jQuery. I have discussed it with faris. he said to fix it.

Reply:
Removed jQuery hide().